scripts/cleanup-test-env.sh

```diff
@@ -19,7 +19,7 @@ set -Eeou pipefail
 projectToSkip="${PROJECT_TO_NOT_DELETE:-NONE}"
 
 # Get all project Ids inside the organization
-projects=$(atlas project ls -o json)
+projects=$(atlas project ls --limit 500 -o json)
```

Comment:
* Do we have a risk of increased latency and timeout from this? 
* What is the rationale in changing it? Is there any failure in our account right now?

Reply:
no latency problem, it's because we're getting errors:  "MAX_GROUPS_PER_ORG_EXCEEDED") Maximum number of groups per organization

Reply:
regarding improving the cleanup process we do have INTMDB-1123 that contains more context and suggestions in case this is helpful

Reply:
thx, as discussed offline I'll just do here the private endpoints deletion

Reply:
yup, I remember we had INTMDB-1123 in our RFW and that's why I was asking since this PR wasn't attached to that. Thank you @AgustinBettati for the note and thanks @lantoli for leaving a comment in https://jira.mongodb.org/browse/INTMDB-1123